### PR TITLE
Enabled passing multiple parameters to fizz_buzz at once

### PR DESCRIPTION
**Pull request by katemildner**

### CODE REVIEW

File: FizzBuzz - multiple parameters.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          input = input.floor
+        end
+      end
+    end
+    print_fizz_buzz(input)
+  end
+end
+
+array_fb = []
+repeat = "y"
```
Comment (ktopolski) [inline FizzBuzz - multiple parameters.rb]: setting variable - in this case `repeat` - before first `while` loop evaluation will work, but there's another loop that might accomplish it without setting variable before loop :)

Please research on `while` loop vs `do while` loop and look up how to write `do while` in Ruby :)

**Review by ktopolski — APPROVED** [review]
Hey Kate,

Job well done, multiparam version of fizz buzz looks really nice!

I left you one thing to research on. It's one of the first things in CS classes when taking on `while` loops. I'm sure this research will benefit you in the future :).

Cheers,
Karol

File: FizzBuzz - multiple parameters.rb (hunk 1 above)
Comment (katemildner) [reply FizzBuzz - multiple parameters.rb]: Thanks for the suggestion - it sounds interesting. I'll educate myself on the topic and will come back with that part redesigned.

**Review by ktopolski — APPROVED** [review]
Hey Kate,

Good job on changing `while` to `do-while`, always nice to see people doing their reading :).

Approving and merging!

**Pull request merged**